src/components/AnnotatedItemField/index.jsx

```diff
@@ -71,29 +78,58 @@ class AnnotatedItemField extends React.Component {
     this.props.editField(file.url);
   }
 
-  handleChange(event) {
-    this.props.editField(event.target.value);
-  }
-
-  handleRichTextChange(value) {
-    this.props.editField(value);
+  editText(text) {
+    if (!text) {
```

Comment:
`// Implicitly clear the content and annotations if the user has deleted all of their text.`